.github/workflows/integration_test.yml

```diff
@@ -16,25 +16,23 @@ jobs:
       - uses: actions/setup-go@main
         with:
           go-version: 1.18
+          
       - name: install kubectl
         run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+
       - name: start local k8s cluster
         run: make cluster-up
         env:
             CLUSTER_PROVIDER: ${{matrix.kube_provider}}
-      - name: Checkout kube-operator repo
```

Comment:
wait a min @marceloamaral , @rootfs 
1st of all, I don't mind we switch to prometheus operator instead of an all in one deployment.
but, some thing I wanna to make it clear.

> To deploy the prometheus operator on the kind cluster, we need to load the images into the cluster.

- [ ] Kind as a k8s cluster will download the image from docker registry no matter local or public right?
- [ ] if some pod waiting for download, via this pr, we should know ... and if kind not download the docker image, we are expecting the CI fails right?

with the my personal confusing points as above, I did a test as https://github.com/sustainable-computing-io/kepler/actions/runs/3118522667/jobs/5057872188
and it seems that if we remove the `_load_prometheus_operator_images_to_local_registry` function invoke, the kind cluster is still able to download the images and start ... correctly?

Reply:
Hence, in this pr, do we have covered any check points to ensure we can see some/any error in test by verifying if all pods were created successfully?
> We don't see errors in the test because the test is not verifying if all containers were created successfully....

Reply:
>Kind as a k8s cluster will download the image from docker registry no matter local or public right?

No. You must push the images to the kind cluster....
I tried many different ways to enable kind download the image from public repository without success...:/

Reply:
> if some pod waiting for download, via this pr, we should know ... and if kind not download the docker image, we are expecting the CI fails right?

Yes! But the old integration test was not verifying if the container were successfully created, so we did not see any error...

Reply:
> with the my personal confusing points as above, I did a test as https://github.com/sustainable-computing-io/kepler/actions/runs/3118522667/jobs/5057872188
and it seems that if we remove the _load_prometheus_operator_images_to_local_registry function invoke, the kind cluster is still able to download the images and start ... correctly?

I made a comment in this PR :)